src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -474,6 +484,58 @@ export abstract class SimpleFindReplaceWidget extends Widget {
 
 		this._innerReplaceDomNode.appendChild(this._replaceBtn.domNode);
 		this._innerReplaceDomNode.appendChild(this._replaceAllBtn.domNode);
+
+		this._resizeSash = this._register(new Sash(this._domNode, { getVerticalSashLeft: () => 0 }, { orientation: Orientation.VERTICAL, size: 2 }));
+
+		this._sashListener.add(this._resizeSash.onDidStart(() => {
```

Comment:
I would only use a DisposableStore when I am managing some disposables that have a different lifecycle from the object that owns them. Here you have some listeners that are added once and should be disposed when the find widget is disposed. So you can get rid of `_sashListener` and just use `this._register` for the listeners